vertica_python/vertica/cursor.py

```diff
@@ -224,6 +237,8 @@ def execute(self, operation, parameters=None, use_prepared_statements=None,
         return self
 
     def executemany(self, operation, seq_of_parameters, use_prepared_statements=None):
+        # type: (str, Sequence[Union[List[Any], Tuple[Any]]], Optional[bool]) -> None
```

Comment:
seq_of_parameters can also be a list/tuple of dicts: `[{'a': 2, 'b': 'bb'}, {'a': 3, 'b': 'foo'}, {'a': 4, 'b': 'xx'}, {'a': 5, 'b': 'bar'}]`

Reply:
Thanks for the hint @sitingren! Fixed now